clayui.com/gatsby-browser.js

```diff
@@ -3,7 +3,20 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-require('./src/styles/main.scss');
+const showAtlas = localStorage.getItem('clay.showAtlas');
+const showSiteCss = localStorage.getItem('clay.showSiteCss');
+
+const devEnv = process.env.GATSBY_CLAY_NIGHTLY === 'true';
+
+if (!devEnv || showAtlas === 'true' || showAtlas === null) {
+	require('@clayui/css/src/scss/atlas.scss');
+} else {
+	require('@clayui/css/src/scss/base.scss');
+}
+
+if (!devEnv || showSiteCss === 'true' || showSiteCss === null) {
+	require('./src/styles/main.scss');
+}
```

Comment:
Hey @bryceosterhaus, it looks like the pattern is showing only the base. Probably due to the above conditional that compared with the same value, I think `showAtlas` and `showSiteCss` is coming undefined.